src/Symfony/Component/Link/README.md

```diff
@@ -0,0 +1,14 @@
+Link Component
+==============
+
+The Link component manages links between resources. It is particularly useful to advise clients
+to preload and prefetch documents through HTTP and HTTP/2 pushes.
```

Comment:
if the component follows some sort of standard, it could be worth mentioning it here